### ISSUE TITLE
Join command not recognizing two keys [Windows 10 Powershell]

### ISSUE THREAD

**Issue by goleary** [mbloch/mapshaper#369]
I'm attempting to join some csv data to a shape file and can't get past this error:
`[join] Error: Expected two key fields: a target field and a source field`
I'm specifying two keys in my command but mapshaper seems to not recognize this.  If I submit only one key (or 3) I get the same error

![image](https://user-images.githubusercontent.com/16123225/66939701-7081e680-eff8-11e9-975e-8bb3a75b6e7d.png)

Environment:
Windows 10
Powershell
mapshaper 0.4.137

**Comment by goleary** [comment]
interestingly enough, upon further investigation, using Command Prompt on windows functions properly, but PowerShell does not:
![image](https://user-images.githubusercontent.com/16123225/66940954-c6f02480-effa-11e9-90f2-04ca5a11d282.png)

**Comment by goleary** [comment]
That works too!  guess it's just a nuance of PowerShell that I was not aware of.

**Comment by mbloch** [comment]
I tried to reproduce this problem on Windows 10 with Powershell versions 5.1 and 6.2 and Node v10.16.3. The -join command (and other commands with comma-separated arguments) worked fine for me without  putting quotes around the key values.

I have a hunch that the error you saw was related to Powershell converting the comma-separated string to an array object. Since we found a workaround for you, and I can't reproduce the issue myself, I'm going to close this issue. Thanks for reporting this :)
